blocks/init/src/Blocks/components/load-more/assets/index.js

```diff
@@ -0,0 +1,25 @@
+/* global esBlocksLocalization */
+
+import domReady from '@wordpress/dom-ready';
+import manifest from './../manifest.json';
+
+domReady(() => {
+	const {
+		componentJsClass,
+	} = manifest;
+
+	const selectors = `.${componentJsClass}`;
+	const elements = document.querySelectorAll(selectors);
+
+	if (elements.length) {
+		import('./load-more').then(({ LoadMore }) => {
+			const loadMore = new LoadMore({
+				triggerElements: elements,
+				ajaxHandler: 'dynamic_data',
+				ajaxUrl: esBlocksLocalization?.ajaxurl,
```

Comment:
Why are we using AJAX for load more functionality instead of using REST API?

REST API is better suited for this (security, performance, etc.)